src/edie/evaluator.py

```diff
@@ -116,16 +120,29 @@ def metadata_evaluation_report_as_dataframe(self):
     def entry_report(self, dictionary_id):
         return self.report['dictionaries'][dictionary_id]['entry_report']
 
+    def visualize(self):
```

Comment:
This function is never called.

Reply:
Yes, this function is experimental, and is intentionally not called in the main function. 
I might refactor/extend/delete this function as we progress and arrive on a consensus about the visualisation part.